vignettes/custom_appearance.Rmd

```diff
@@ -90,3 +94,239 @@ lyt3 <- basic_table() %>%
 tbl3 <- build_table(lyt3, ex_adsl)
 tbl3
 ```
+
+### Top-left Materials
+
+The sequence of strings printed in the area between the column header display and the first row label \emph{top left material} can be modified during pre-processing within the table constructor `basic_table()` using `append_topleft`, the label position in row splits `split_rows_by` and during post-processing using the `top_left()` function. Note: No structure or indenting is applied to these strings.
```

Comment:
Huh, I could have sworn I put it in `basic_table`. I guess I just at one point intended to. Thats my mistake, apologies.

That said, " Note: No structure or indenting is applied to these strings." is not fully true, indenting is automatically added *only in the case of `label_pos="topleft"`* (which as I mentioned before is the most canonical usage imo (and so should be listed first).

Please revise to reflect that